plugins/search/src/components/SearchResult/SearchResult.tsx

```diff
@@ -48,7 +52,24 @@ const SearchResultComponent = ({ children }: Props) => {
     return <EmptyState missing="data" title="Sorry, no results were found" />;
   }
 
-  return children({ results: value.results });
+  const handlePageChange = useCallback(
+    (_: React.ChangeEvent<unknown>, pageNum: number) => {
+      setPage({ offset: (pageNum - 1) * pageSize, limit: pageSize });
+      window.scrollTo({ top: 0, left: 0, behavior: 'smooth' });
+    },
+    [setPage, pageSize],
+  );
+
+  return (
+    <>
+      {children({ results: value.results })}
+      <Pagination
```

Comment:
Here is where I have to continue. Offset seems to be off, I still have two pagers, tests and some cleanup, but next week

Reply:
@Fox32 I wonder if we should make inclusion of this `<Pagination />` component optional, based on a prop or something?

The `<SearchResult />` component is intended to be kind of a utility that makes accessing the results of a search in the current context easy.  But it's not a given that every use of search would be paginated (in fact, the only one might be the search page).

It should definitely be included here, I just think it should just be possible to toggle whether or not it gets included.